src/main/scala/is/hail/variant/VariantSampleMatrix.scala

```diff
@@ -925,7 +928,7 @@ class VariantSampleMatrix[T](val hc: HailContext, val metadata: VSMMetadata,
         case other =>
           fatal(
             s"""method 'annotate_variants_table' expects a key table keyed by one of the following:
-               |  [ Variant ]
+               |  [ $vSignature ]
```

Comment:
Ah.  Yes!  In my coming code, Locus is not hardcoded, but Interval still is.  I think if we make Interval type parameterized: Interval[T], then everything generalizes nicely.